tests/cross_functional/performance/io_workload/test_fio_benchmark.py

```
@@ -481,19 +485,41 @@ def test_fio_workload_simple(self, interface, io_pattern):
 
     @skipif_ocs_version("<4.6")
     @pytest.mark.parametrize(
-        argnames=["io_pattern", "bs", "cmp_ratio"],
+        argnames=["io_pattern", "bs", "cmp_ratio", "erasure_coded"],
```

Ruff (0.15.14)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)